server/src/main/resources/reference.conf

```diff
@@ -78,3 +79,27 @@ overrides {
     }
   }
 }
+
+staging {
+  application {
+    # Additional HTTPS security headers.  You can add, modify, remove as needed to enhance
+    # the security profile of our Kotlin based applications.
+    httpsHeaders: ["X-Content-Type-Options: nosniff",
```

Comment:
if you want to dedup this you can put it in a block above and reference it, but up to you